src/commands/test/utils.py

```diff
@@ -86,7 +86,7 @@ def match(self, state_event: Event) -> bool:
 @dataclass
 class TestSubject:
     """
-    A dataclass consisting of identification of a single test bundle, and target functions
+    Deprecated. Use `TestSuite`.
```

Comment:
Why do you deprecate that? The case is we use this one in runner, yup?🤔 
You can make type alias instead `TestSubject = TestSuite` then

Reply:
I tried initially, but it `TestSuite::test_functions` has different type than `TestSubject::test_functions`.